mass/core/query_handler.py

```diff
@@ -14,7 +14,12 @@
 # limitations under the License.
 #
 
+
 """Contains implementation of a QueryHandler to field queries on metadata"""
+from typing import Dict, List, Set
```

Comment:
Using Dict, List, and Set is deprecated in python3.9 instead you can use the building dict, list, set with squared brackets for type annotation.